Convert source tree from UTF8 chars into us-ascii

Hi,
  java-8 is picky about UTF8 chars in the sources, it even bother about such comment in comments. Here are some I hit in current git checkout.


```
/scratch/var/tmp/portage/dev-java/htsjdk-9999/work/htsjdk-9999/src/test/java/htsjdk/samtools/util/PositionalOutputStreamTest.java:4: error: unmappable character for encoding ASCII
 * Copyright (c) 2015 Daniel G??mez-S??nchez
                              ^
```

--- a/src/test/java/htsjdk/samtools/util/PositionalOutputStreamTest.java
+++ b/src/test/java/htsjdk/samtools/util/PositionalOutputStreamTest.java
@@ -1,7 +1,7 @@
 /*
  * The MIT License (MIT)
  *
- * Copyright (c) 2015 Daniel Gómez-Sánchez
+ * Copyright (c) 2015 Daniel Gomez-Sanchez
  *
  * Permission is hereby granted, free of charge, to any person obtaining a copy
  * of this software and associated documentation files (the "Software"), to deal

